src/backend/egl/display.rs

```diff
@@ -517,6 +517,28 @@ impl EGLDisplay {
         &self.dmabuf_import_formats
     }
 
+    /// Returns when the display supports extensions required for smithays
+    /// damage tracking helpers.
+    pub fn supports_damage(&self) -> DamageSupport {
```

Comment:
To the api user, this just needs to be a boolean? Interally we would need this enum still

Reply:
Done.  enum is still public, because it is part of the `NativeSurface` trait signature